README.md

```diff
@@ -30,14 +30,16 @@ The default directory for mosaic to store chain data is `~/.mosaic`.
 You can specify a different directory with the `--mosaic-dir` option.
 
 Examples:
-* Starts three containers to follow these chains:
-  * `./mosaic start 200 ropsten rinkeby`
+* Starts five containers to follow these chains:
+  * `./mosaic start ropsten rinkeby 1406 1407 1414`
```

Comment:
Hey, sorry this is my fault, but rinkeby cannot be used. Can you please remove rinkeby here?